### DISCUSSION TITLE
Is there more informations about tracker

### DISCUSSION THREAD

**Discussion by jeffyjf** [anacrolix/torrent#840] [Q&A]
I plan  to implement a bittorrent tracker srver with go. I find that there are tracker's base code in the repository. But, I can't find any other informations about these codes.  Is there more informations about these tracker codes (such as: documentation, test codes, sample implemention etc.)?

Thanks.

**Comment by anacrolix** [comment]
There are some parts in anacrolix/torrent that can be pulled together to make a tracker, but my tracker implementation is currently private. The question has been asked before, but I don't know if I am interested in sharing that part right now.

**Reply by anacrolix** [reply]
You can make use of tracker.server.AnnounceHandler if you bring implementations for some of hte interfaces there. That's how my downstream implementation works.

**Reply by jeffyjf** [reply]
Got it, thanks.

**Reply by anacrolix** [reply]
@jeffyjf I'm considering open sourcing my tracker implementation, does that interest you?

**Reply by LeventeLantos** [reply]
@anacrolix Yes that would be awesome!

**Reply by philrhc** [reply]
also interested
